src/refactorings/convert-let-to-const/convert-let-to-const.test.ts

```diff
@@ -0,0 +1,160 @@
+import { Editor, ErrorReason, Code } from "../../editor/editor";
+import { Selection } from "../../editor/selection";
+import { InMemoryEditor } from "../../editor/adapters/in-memory-editor";
+import { testEach } from "../../tests-helpers";
+
+import { convertLetToConst } from "./convert-let-to-const";
+
+describe("Convert Let To Const", () => {
+  let showErrorMessage: Editor["showError"];
+
+  beforeEach(() => {
+    showErrorMessage = jest.fn();
+  });
+
+  testEach<{ code: Code; selection?: Selection; expected: Code }>(
+    "should convert let to const",
+    [
+      {
+        description: "non-mutated variable declared not in a block",
+        code: `let someVariable = 'value';`,
+        expected: `const someVariable = 'value';`
+      },
+      {
+        description: "non-mutated variable declared in a block",
+        code: `{
+  let someVariable = 'value';
+}`,
+        expected: `{
+  const someVariable = 'value';
+}`
+      },
+      {
+        description: "multiple non-mutated variables declared together",
```

Comment:
Thinking about this one, I kind of overlooked the fact that the refactor would leave us with invalid code due to the `'const' declarations must be initialized.` error.

At least initially I'm going to prevent the refactor for this case.

Eventually, I think the solution for this is to move the selected variable onto its own line leaving the other variable unaffected by the refactor.

@nicoespeon does that feel like a reasonable decision?

Reply:
Yep, I think it's the best decision. We shouldn't turn an un-initialized `let` into `const` or that would be invalid. Of course it makes the multi-variables trickier to deal with, but it's the right thing to do 👍